cmd/proxy/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+	"strings"
+
+	"github.com/openshift/elasticsearch-proxy/pkg/config"
+	auth "github.com/openshift/elasticsearch-proxy/pkg/handlers/authorization"
+	cl "github.com/openshift/elasticsearch-proxy/pkg/handlers/clusterlogging"
+	"github.com/openshift/elasticsearch-proxy/pkg/proxy"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	serviceAccountTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
should this actually be configurable via CLI?

Reply:
+1 - if for no other reason than it makes testing much easier

Reply:
I need to check but this may no longer be relevant with the move to use client-go and it relies on this internally.

Reply:
hmm - that's probably worse then, if the path `/var/run/...` is hardcoded there and cannot be changed . . .

Reply:
removed.  no longer required. this does not address @richm concern about testing

Reply:
not sure, but it looks like in the operator-sdk test code it tries to figure out if you are running in a real pod or if you are passing in a `KUBECONFIG` from your test env